org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/CompletionHandlerTest.java

```diff
@@ -2616,6 +2617,58 @@ public void testStaticImports2() throws Exception {
 		}
 	}
 
+	@Test
+	public void testStarImports() throws Exception {
+		List<String> favorites = new ArrayList<>();
+		favorites.add("java.lang.Math.*");
+		Preferences prefs = PreferenceManager.getPrefs(null);
+		List<String> oldFavorites = Arrays.asList(prefs.getJavaCompletionFavoriteMembers());
+		int onDemandThreshold = prefs.getImportOnDemandThreshold();
+		int staticOnDemandThreshold = prefs.getStaticImportOnDemandThreshold();
+		prefs.setJavaCompletionFavoriteMembers(favorites);
+		prefs.setImportOnDemandThreshold(2);
+		prefs.setStaticImportOnDemandThreshold(2);
+		try {
+			ICompilationUnit unit = getWorkingCopy("src/test1/B.java",
+			//@formatter:off
+			"package test1;\n" +
+			"import static java.lang.Math.sqrt;\n" +
+			"import java.util.List;\n" +
+			"public class B {\n" +
+			"....List<String> list = new ArrayL\n" +
```

Comment:
`....`  ??? 

The test fails on jenkins

Reply:
Fixed.